src/js/view/reducers/logViewerReducer.js

```diff
@@ -27,12 +35,24 @@ export const logViewerReducer = (state = initialState, action) => {
     case 'LOGVIEWER_SET_LOG': {
       console.log('SETTING');
       const { sourcePath, log, startByteOfLines } = action.data;
+      let meanByteValueOfInitialLines = calculateMeanValueOfBytesPerLine(
+        startByteOfLines,
+        sourcePath
```

Comment:
Should sourcePath be here?

Reply:
Nope, thanks for finding that. Used it while debugging and forgot to erase it.